Backpack/src/main/java/net/skyscanner/backpack/calendar/view/CalendarView.kt

```diff
@@ -31,19 +31,21 @@ import net.skyscanner.backpack.calendar.presenter.MonthAdapter
 /**
  * This displays a list of months in a calendar format with selectable days.
  */
-internal class CalendarView constructor(context: Context, attr: AttributeSet?) : ListView(context, attr),
-  OnScrollListener {
-
-  override fun onScrollStateChanged(view: AbsListView?, scrollState: Int) {
-    // Do nothing
-  }
+internal class CalendarView constructor(
+  context: Context,
+  attr: AttributeSet?
+) : ListView(context, attr), AbsListView.OnScrollListener {
 
   private var scrollFriction = 1.0f
   private var selectedDay: CalendarDay = CalendarDay()
   private var previousScrollPosition: Long = 0
   private var previousScrollState = OnScrollListener.SCROLL_STATE_IDLE
   private var currentScrollState = OnScrollListener.SCROLL_STATE_IDLE
 
+  override fun onScrollStateChanged(view: AbsListView?, scrollState: Int) {
+    // TODO
```

Comment:
Why did this became a TODO? I don't think we expect any behavior on scrollStateChange, hence it contained a comment before. Are we planning to do anything here?

Reply:
Are we not planning to add the same functionality present on IOS? That shoes the current year at the top, over the calendar? That's why I changed to TODO. If we don't plan to do anything with this it can be just removed. But I believe we do plan to add this feature

Reply:
I think we do -- I believe Stark is just about to start doing that. We can keep it, just didn't understand that change belonging to this PR.